routes/buyer/buyer.js

```diff
@@ -136,6 +136,97 @@ exports.updateBuyerById = function(req, res) {
     }
 };
 
+
+/**
+
+ Update Buyer Demand
+
+
+ **/
+
+exports.editDemand = function(req, res) {
+    if(common.isAuthenticated(req, res)){
+
+        Demand.findByIdAndUpdate({_id:req.params.demand_id}, req.body, function (err, changes) {
+            console.log(err, changes);
+            if (err) {
+                handleDBError(err, res);
```

Comment:
@nick22891 i believe this line may have an error. The handleDBerror should be called by prefixing common before the function name.